### DISCUSSION TITLE
Lab Assignment #6

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#148] [Assignments]
Paste your GitHub pages URL for Week_06 as a comment below to submit your [lab assignment #6](https://github.com/albertkun/21S-ASIAAM-191A/blob/main/Week_7/Lab/lab_assignment.md).

e.g. https://akochaphum.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by skarakas1** [comment]
https://skarakas1.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Reply by albertkun** [reply]
@skarakas1 nice css and early submission!

**Reply by skarakas1** [reply]
Thank you!

**Comment by alizezila** [comment]
https://alizezila.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Reply by skarakas1** [reply]
Hey I think there may be a problem with your basemap, I know some of them won't work without API keys. The code that you copy and paste should say if it needs a basemap or not if that's the issue. Also--cute dog!

**Reply by alizezila** [reply]
@skarakas1 oh no 😣 I will look into it, thank you for the heads up!

**Reply by alizezila** [reply]
Fixed hehe <3

**Comment by ghost** [comment]
https://ktgreenler.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by epham36** [comment]
https://epham36.github.io/21S-ASIAAM-191A-Assignments/Week_07/

**Comment by jason-muljadi** [comment]
https://jason-muljadi.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by rachan2023** [comment]
https://rachan2023.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by KenNL42** [comment]
https://kennl42.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Reply by alizezila** [reply]
Idk why I just saw this, but this was a great idea! Budget geoguesser is so creative 🤣

**Comment by lvalles10** [comment]
https://lvalles10.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by vcta99** [comment]
https://vcta99.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Reply by rh5140** [reply]
Vanessa, I love the concept of your site and, of course, the pun!!! It's so cute!!!

I noticed that the circle markers aren't different colors like you intended them to be. (I may have been struggling a lot with this and looked at the code in every previous submission to try to figure out what I was doing wrong...)

You would need to add circleOptions into addLayer, like
`L.circleMarker([data.lat,data.lng], circleOptions)`

Also, you're missing the hashtag symbol for one of the hex codes (800000)

**Reply by vcta99** [reply]
oh my god Ray thank you so much for the circle markers tip. I, like you, struggled like crazy with it, will update the code now~

**Comment by rh5140** [comment]
https://rh5140.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

(My bad for procrastinating too much, but for some reason I'm not able to change the fill color of the circles and I'm super confused because I straight up copypasted code that worked for other people to make sure... too tired to puzzle it out though zzzz)

EDIT: Thanks Sam!! Turned out I was missing circleOptions in addLayer!!

**Reply by skarakas1** [reply]
I like your color scheme Ray it's very watermelon-y!

**Comment by jsxu315** [comment]
https://jsxu315.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by okcirE** [comment]
https://okcire.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by latishasumardy** [comment]
https://latishasumardy.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Reply by skarakas1** [reply]
I love the design of your site, so clean!

**Comment by Wny-Duong** [comment]
https://wny-duong.github.io/21S-ASIAAM-191A-Assignments/Week_07/

**Comment by annyrviloria** [comment]
https://annyrviloria.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by ninalong** [comment]
https://ninalong.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by avaitt** [comment]
https://avaitt.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html

**Comment by nnhimichelle** [comment]
https://nnhimichelle.github.io/21S-ASIAAM-191A-Assignments/Week_07/index.html sorry this is so late!

**Comment by brillianz** [comment]
https://brillianz.github.io/21S-ASIAAM-191A-Assignments/Lab_04/index.html

**Reply by albertkun** [reply]
wow looks good! @brillianz
